### PR TITLE
Revamp Community tab and put Blog on main navbar

### PR DESCRIPTION
**Pull request by GregSutcliffe**
From the view of someone new to the project, and wanting to know more
about what to watch, how to get help, or how to contribute, I think it
makes sense to have all support / media / code & non-code contributions
on a single page.

Regarding the earlier pull request to move the blog to the front page, I've done this again because:
- there's more content there now
- the blog is now getting more traffic than the media page (according to Google Analytics, its 1.93% vs 1.23% of unique pageviews)
- The Atom feed links to "http://theforeman.org". This can't be changed (jekyll-feed uses the :url: parameter in _config.yml) so we should make it easy for people coming to the site from RSS to get back to the blog.

I've also re-ordered the support section - a new user pointed out that based on the current order, one might expect the mailing-list to be more responsive than IRC, when actually the reverse is true.

Comments?

### CODE REVIEW

File: support.md [hunk 1]
```diff
@@ -1,89 +0,0 @@
...
-
-## Gathering information
-
-To get appropriate support, you need to provide sufficient information about
-your operating system, environment and setup. You can utilize the
-foreman-debug tool - for more info see the [manual](/manuals/latest/index.html#7.3GettingHelp).
-
-## ![Redmine icon](static/images/redmine.png) [Issue tracker](http://projects.theforeman.org/projects/foreman/issues)
-
-The issue tracker (Redmine) should be used to check for existing issues or to file a new issue.  First, [find the project](http://projects.theforeman.org/projects) and then use the _Issues_ tab to view bugs and feature requests.  Use the search feature at the top of the issue list to filter by _Category_ to narrow down the results.
```
Comment (domcleal) [inline support.md:17]: All of this (new) information has been removed?

**Comment by domcleal** [comment]
I don't think all of that information should be left on one page, it doesn't look good.  There's a huge amount on it, everything from how to start up a Rails server to IRC guidelines, it doesn't make much sense?

Still don't think the blog should be a top-level item compared to say, support information.  Perhaps use a menu.

File: support.md (hunk 1 above)
Comment (GregSutcliffe) [inline support.md:17]: Whoops, I'll fix that. I was diffing an old branch.

**Comment by GregSutcliffe** [comment]
I like the idea of a menu for the differing parts of the (current) large page - I'll see what I can do with my meagre CSS skills.

I do feel the blog needs to be prominent. We seem to be at an impasse on this - I'll take it to the dev list for more voices to resolve it.

**Comment by domcleal** [comment]
If there's a menu then there's no real need for a separate top-level link, it can simply go under that.

**Comment by domcleal** [comment]
It occurred to me that we've moved away from menus towards the landing page style shown in documentation.  Would something similar make more sense instead?  One page with links to IRC, mailing list, Redmine, and a variety of contributing pages (one for code contributions, one for translating etc).

**Comment by GregSutcliffe** [comment]
That would possibly work for contributing - I'm still concerned about the blog/video/slides etc. Why did we move away from menus in the first place? I think some people might get lost if they don't find what they're looking for on the frontpage, but perhaps we can find a middle ground...

**Comment by dLobatog** [comment]
I think what @domcleal suggested is a better idea, as well as moving media to the `Documentation` menu. In the end that's what media is, a kind of documentation :smile:

**Comment by GregSutcliffe** [comment]
I think @domcleal suggested a page _like_ Documentation, with further links contained, rather than adding to the existing page - that page is busy enough already, and I'm hearing quite a few people saying it takes them a while to find what they want on it.

I'm still trying to understand the arguments against menus, as I generally find them useful personally - I can peruse potential next steps without leaving the homepage, rather than many page loads hunting for something.

**Comment by Moncky** [comment]
I think that a combination of what @GregSutcliffe and @domcleal are suggesting is the best way forward.  The blog needs to be on the front page, or at least a latest news with the top 4 or 5 blog posts linked.

For an existing user its in some ways fine to have it buried a couple of clicks away from the home page, but for a prospective or new user it would be much clearer if it were on the front page and accessible with only one click.  A quick and very unscientific survey of a couple of other opensource sites all have either a blog link or a latest news section on their homepage.  (Debian, Apache, Puppet, CentOS, Postgres, Mongo)

The community tab in general could be better served with a landing style page with sections for Communication (IRC, Mailing list info), Presentations (Slides, Videos), Contributing (Non-Code Contributions, Code Contributions), Event Info (Conferences, future Webinars etc)

Additionally the Code Contributions page should also be linked on the Documentation tab as its valid to look in both those locations for that info.

Finally (whilst I appear to be on a roll), Big Foreman community run events such Foreman Construction Day should have a prominent, temporary banner presence on the homepage, and possibly the landing pages, so that interested parties that may not be at the point of monitoring mailing lists/twitter/blog etc can see whats going on.

**Comment by dLobatog** [comment]
Any updates? What @Moncky suggests above sounds sensible to me

**Comment by GregSutcliffe** [comment]
@dLobatog sorry, this dropped off my radar. I have a bunch of reports to work on this week, but I'll try to work on something next week that reflects the comments in the PR so far. Will ping you when it's done ;)

**Comment by GregSutcliffe** [comment]
#634 has added the Blog link to the frontpage - I'll re-write this PR regarding the contribution section in the near future.

**Comment by mmoll** [comment]
closing after inactivity.

**Pull request closed without merging**